pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -700,6 +703,85 @@ var _ = Describe("ImageUpload", func() {
 			validatePVCDefaultInstancetypeLabels()
 		})
 
+		It("Should create DataSource pointing to the PVC", func() {
+			testInit(http.StatusOK)
+			cmd := clientcmd.NewRepeatableVirtctlCommand(
+				commandName, "dv", targetName,
+				"--size", dvSize,
+				"--uploadproxy-url", server.URL,
+				"--insecure",
+				"--force-bind",
+				"--datasource",
+				"--image-path", imagePath,
+				"--default-instancetype", "fake.large",
+				"--default-instancetype-kind", "fake.large",
+				"--default-preference", "fake.centos",
+				"--default-preference-kind", "fake.centos",
+			)
+			Expect(cmd()).To(Succeed())
+			Expect(pvcCreateCalled.IsTrue()).To(BeTrue())
+			Expect(kvtesting.FilterActions(&cdiClient.Fake, "create", "datasources")).To(HaveLen(1))
```

Comment:
This could be dropped because it is covered by the assertions below.

Reply:
Sure let's drop it.